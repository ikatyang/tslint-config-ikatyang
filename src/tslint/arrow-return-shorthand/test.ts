// good

const good1 = x => x;
const good2 = x => x
  .long
  .long
  .long
  .long;

// bad

const bad1 = x => { return x };
const bad2 = x => {
  return x
    .long
    .long
    .long
    .long;
}
