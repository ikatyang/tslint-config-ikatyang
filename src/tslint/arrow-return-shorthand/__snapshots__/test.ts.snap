// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
// good

const good1 = x => x;
const good2 = x => x
  .long
  .long
  .long
  .long;

// bad

const bad1 = x => { return x };
                  ~~~~~~~~~~~~  [This arrow function body can be simplified by omitting the curly braces and the keyword 'return'.]
const bad2 = x => {
                  ~
  return x
~~~~~~~~~~
    .long
~~~~~~~~~
    .long
~~~~~~~~~
    .long
~~~~~~~~~
    .long;
~~~~~~~~~~
}
~ [This arrow function body can be simplified by omitting the curly braces and the keyword 'return'.]

`;
